WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsTrackerWrapper.kt

```diff
@@ -16,7 +21,32 @@ class AnalyticsTrackerWrapper
     }
 
     fun track(stat: Stat, feature: FeatureConfig) {
-        AnalyticsTracker.track(stat, mapOf(feature.remoteField to feature.isEnabled()))
+        AnalyticsTracker.track(
+                stat,
+                mapOf(
+                        feature.remoteField to feature.isEnabled(),
+                        "${feature.remoteField}_state" to feature.featureState().toName()
+                )
+        )
+    }
+
+    fun track(stat: Stat, remoteField: String, featureState: FeatureState) {
+        AnalyticsTracker.track(
+                stat,
+                mapOf(
+                        remoteField to featureState.isEnabled,
+                        "${remoteField}_state" to featureState.toName()
+                )
+        )
+    }
```

Comment:
I think we can probably extract a common logic between this two methods?

Reply:
done in a752e10

Reply:
Fixed in a752e10